tensorflow/lite/ios/ios.bzl

```diff
@@ -78,6 +79,59 @@ def tflite_ios_framework(
         ],
     )
 
+# iOS xcframework with symbol allowlist. Exported C++ symbols might cause symbol
+# collision with other libraries. List of symbols to allowlist can be
+# generated by running `nm -m -g FRAMEWORK_LIBRARY | grep _TfLite` for framework
+# built with `ios_static_framework` rule.
+def tflite_ios_xcframework(
+        name,
+        bundle_name,
+        allowlist_symbols_file,
+        **kwargs):
+    """Apply symbol hiding to the output of ios_static_framework.
+
+    Args:
+      name: The name of the target.
+      bundle_name: The name to give to the framework bundle, without the
+          ".framework" extension. If omitted, the target's name will be used.
+      allowlist_symbols_file: a file including a list of allowed symbols,
+          one symbol per line.
+      **kwargs: Pass-through arguments.
+    """
+
+    preprocessed_name = "Preprocessed_" + name
+    apple_static_xcframework(
```

Comment:
This builds xcframework with static libraries, is this what you want? btw we are already releasing xcframeworks to cocoapods and you can download the xcframeworks through the download link https://github.com/CocoaPods/Specs/blob/master/Specs/1/6/0/TensorFlowLiteC/0.0.1-nightly.20220728/TensorFlowLiteC.podspec.json.

Reply:
> This builds xcframework with static libraries, is this what you want?

Yes, I'm pretty much just mirroring the previous build setup but producing xcframeworks instead. Since the official binaries are still static as well, I assume that's still recommended as well?

> btw we are already releasing xcframeworks

Yea we are building with some custom flags which is why we've been building directly